### DISCUSSION TITLE
Nucleo dtsi not such file or directory.

### DISCUSSION THREAD

**Discussion by RaysoftTechnologiesInc** [zephyrproject-rtos/zephyr#61878] [General] [answered]
Hi,

I am a bit new to Zephyr and I am currently trying to run  a blinky on my nucleo board, wl55jc, However I am currently getting this error
![image](https://github.com/zephyrproject-rtos/zephyr/assets/45296244/50120abd-5425-47c5-84df-621c20246488)

I have tried running west update but it is not changing anything.

Could you please assist?

**Comment by henrikbrixandersen** [comment]
The file in question is provided by the `hal_st` module. Please ensure your manifest contains this module before running `west update`:

```
$ west list hal_st
hal_st       modules/hal/st               9b128caf3e7b2e750169b880e83f210ea2213473 https://github.com/zephyrproject-rtos/hal_st

$ git describe 
zephyr-v3.4.0-2797-g075a859869

$ ls -l ../modules/hal/stm32/dts/st/wl/stm32wl55jcix-pinctrl.dtsi 
-rw-r--r-- 1 hebad hebad 20367 May 10 11:12 ../modules/hal/stm32/dts/st/wl/stm32wl55jcix-pinctrl.dtsi


```

**Reply by erwango** [reply]
Actually, it's provided by the `hal_stm32` module.
`hal_st` module provides support for ST components that can be used on other vendors hw, such as MEMS sensors.
To build on STM32 parts, you need the `hal_stm32` module.

**Comment by RaysoftTechnologiesInc** [comment]
Hi,
it seems the module does not exist
![image](https://github.com/zephyrproject-rtos/zephyr/assets/45296244/5639522e-c5e6-4de2-8b50-a4f90ea0cd29)

I have been running the west update, isn't it supposed to update everything?

**Comment by henrikbrixandersen** [comment, marked as answer]
You are not using upstream Zephyr. You are using the NordicSemi downstream SDK (NCS). It only includes the NordicSemi HAL.

**Comment by henrikbrixandersen** [comment]
In the future, please use our bug template when reporting bugs. If you had included the information requested in https://github.com/zephyrproject-rtos/zephyr/blob/main/.github/ISSUE_TEMPLATE/001_bug_report.md this would have been obvious from the start.
